docs/admin/upgrade.rst

```diff
@@ -127,7 +127,7 @@ Notable configuration or dependencies changes:
 * There is change in ``DEFAULT_THROTTLE_CLASSES`` setting to allow reporting of rate limiting in the API.
 * There are some new and updated requirements.
 * There is a change in :setting:`django:INSTALLED_APPS`.
-* The :ref:`deepl` machine translation now defaults to v2 API, you might need to adjust :setting:`MT_DEEPL_API_VERSION` in case your current DeepL subscription does not support that.
+* The :ref:`deepl` machine translation now defaults to v2 API. The ``MT_DEEPL_API_VERSION`` setting has been removed in Version 4.7. You might need to adjust the new :setting:`MT_DEEPL_API_URL` in case your current DeepL subscription does not support that.
```

Comment:
Let’s use

> ...in case your current DeepL subscription does not support v2 API.

It is unclear if `that` refers to the first part of the sentence or the previous sentence.

Reply:
This is a difficult one.
Since we are talking about upgrading from 4.0 to 4.1, I left the information there as it was originally and tried to keep the old sentences. Is anyone ever going to migrate from 4.0 to 4.1 these days? Then the original info would be helpful, but fails the link test, because the 4.7 docs don't contain the target anymore.
The entire bullet point looks awkward to me when I look at it from 4.7.
Perhaps something like:
* The ``MT_DEEPL_API_VERSION`` setting has been removed in Version 4.7. The :ref:`deepl` machine translation now uses the new :setting:`MT_DEEPL_API_URL` instead. You might need to adjust :setting:`MT_DEEPL_API_URL` to match your subsciption.